pegasus/test/test_deliverer_render.rb

```diff
@@ -42,6 +42,8 @@ def run(*args, &block)
   #       - body.html
   #       - body.txt
   def test_deliverer_render_all
+    skip 'Skipping test while staging-next is active.'
```

Comment:
Is it possible to only skip when in the staging-next environment by any chance? Something like `skip if RACK_ENV=staging-next`?

(I have very little knowledge on how our environments work so I have no idea if this is possible...)

Reply:
Unfortunately, `staging-next` uses `RACK_ENV=test` to ensure that it behaves like test.code.org and runs all unit tests and all UI tests.